x-pack/plugins/endpoint/kibana.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "endpoint",
+  "version": "1.0.0",
```

Comment:
should we call it a 1.0.0 or a 0.9.0 or does it matter in any way to us?  @oatkiller

Reply:
I think this just reports a version number in the UI. I don't know who might rely on it. @james-elastic ?

Reply:
I don't think it matters either way but I filed https://github.com/elastic/kibana/issues/51226 to be sure.